### PR TITLE
Integration Tests - Nested Commenting Feature 

### PR DESCRIPTION
**Pull request by thisaltennakoon**

### CODE REVIEW

**Review by tmkasun — APPROVED** [review]

**Comment by tmkasun** [comment]
Let's wait till the github actions complete and merge

**Comment by tmkasun** [comment]
@thisaltennakoon can you please address the conflicts and give an update here?

**Comment by thisaltennakoon** [comment]
Conflicts have been resolved @tmkasun

File: config/bam.xml [hunk 1]
```diff
@@ -1,4 +0,0 @@
-<BamConfig>
```
Comment (tharindu1st) [inline config/bam.xml:1]: why this files are deleted

**Review by tharindu1st — CHANGES_REQUESTED** [review]

**Comment by tharindu1st** [comment]
@thisaltennakoon  pr need to be revisit since pr was removing files

**Comment by thisaltennakoon** [comment]
The same things are included in this [PR](https://github.com/wso2/product-apim/pull/10259) and it has been merged.

**Pull request closed without merging**